src/languages/mariadb/mariadb.formatter.ts

```diff
@@ -268,7 +268,8 @@ export default class MariaDbFormatter extends Formatter {
       reservedLogicalOperators: ['AND', 'OR', 'XOR'],
       reservedKeywords: keywords,
       reservedFunctionNames: functions,
-      stringTypes: ['""', { quote: "''", prefixes: ['X'] }],
+      // TODO: support _ char set prefixes such as _utf8, _latin1, _binary, _utf8mb4, etc.
+      stringTypes: ['""', { quote: "''", prefixes: ['B', 'X'] }],
```

Comment:
Are we sure that MariaDB does not support `B""` and `X""` strings?

Should also document MariaDB `B`-strings in wiki.

Reply:
all the examples use single quotes but double quote strings are mentioned in 1 line in the MariaDB docs
so presumably they are compatible, will update